.gitlab-ci.yml

```diff
@@ -0,0 +1,58 @@
+include:
+  - project: 'analytics/artificial-intelligence/ai-platform/aip-infrastructure/ci-templates/ci-cd-template'
+    ref: &include_ref 'v4'
+    file: 'environments/devex.yml'
+  - project: 'analytics/artificial-intelligence/ai-platform/aip-infrastructure/ci-templates/ci-cd-template'
+    ref: *include_ref
+    file: '/blocks/python.yml'
+
+variables:
+  PY_LIBRARY_NAME: "zillow-kserve"
+  MAJOR_VERSION: "0"
+  MINOR_VERSION: "10"
+  RELEASE_BRANCH: "zillow/release-0.10.2"
+  KSERVE_VERSION_PATH: "python/kserve/setup.py"
+  PUBLISH: "true"
+
+stages:
+  - build
+
+.version: &version |
+  # Extract the open source KServe version as the basis for our forked library version.
+  KSERVE_VERSION=$(cat python/VERSION)
+
+  PY_LIBRARY_VERSION="${MAJOR_VERSION}.${MINOR_VERSION}.${CI_PIPELINE_IID}"
+  if [ "${CI_COMMIT_BRANCH}" != "${RELEASE_BRANCH}" ]; then
+    PY_LIBRARY_VERSION="${PY_LIBRARY_VERSION}.dev${CI_PIPELINE_IID}"
+  fi
+  IMAGE_TAG="${PY_LIBRARY_VERSION}"
```

Comment:
I couldn't find how this image tag is used. Do we still need it?

Reply:
Yeah might not needed anymore for cicd v4